BiweeklyMeetings/2022-08-02.md

```diff
@@ -0,0 +1,323 @@
+
+**Matthias** -: Okay. Because my main understanding on this, or I mean, experience that's... I mean the catalyst voting in general, it's always been done by [inaudible 00:39:01] themselves. So as part of the Wallet interface, usually do things a bit differently that usually Turmo for that in the past, it's pretty nice just acreage transaction registration as part of the Wallet itself. So I don't quite see the need for an external API for that to let any depth to the registration themselves.
+
+**Sebastien** -: Oh, so the reason for this is because they want to have this de-rep system. So basically anybody can propose themselves to be a representative for catalyst voters and you can delegate your aid out to this representative. Similar to stake pools, there needs to be some sort of stake pool viewer just like there needs to be some sort of de-rep reviewer. So there'll be websites that are made for this. And then some of these dealers may have their own website with a click here to delegate me on all those kind of interface. So the motivation for this is almost only the direct feature.
+
+**Matthias** -: Okay. So, then I don't get the API cause a lot of the API are-
+
+**Sebastien** -: Yeah.
+
+**Matthias** -: It's leveraging key derivation path, which suggests that the key belongs to the Wallet and not to some delegated-
+
+**Sebastien** -: Yeah. I think this is a mistake. It should be like other standards where you start with zero and then if zero is huge to get one and the Wallet handles this, I think. Having the debt analysis is going to be a huge disaster.
+
+**Matthias** -: So okay. Definitely a need for some of the motivation and rational. So I've already flagged that on the PR waiting for some comments now on this. Yeah. So I guess we can just leave it there for now. If we have any of update from the author before the next meeting, we can discuss that again in the next meeting otherwise, I just wait.
+
+#### PR299
+[PR299 CIP-0068? | Datum Metadata Standard](https://github.com/cardano-foundation/CIPs/pull/299)
+
+**Matthias** -: To do. Next one on the agenda, CIP 68. Day two metadata standard, which comes as a replacement or an iteration on the existing metadata standards. But this one aims to be a bit general, although the specification is also focused on entities at this stage because that's the main use case driving that proposal.
+
+**Matthias** -: The main spirit as for... I think the main rational as for why put a metadata in the data room instead of in the transaction is because you cannot access metadata of the transaction as part of your produce script, which is a little odd. And it truly sounds like at a political level, at least the transaction metadata and the data room have become kind of redundant with one another. So having access to the metadata inside the script allows for some extra validations to be done in particular when moving NFTs around.
```

Comment:
```suggestion
**Matthias** -: The main spirit as for... I think the main rationale as for why put a metadata in the datum instead of in the transaction is because you cannot access metadata of the transaction as part of your produce script, which is a little odd. And it truly sounds like at a political level, at least the transaction metadata and the data room have become kind of redundant with one another. So having access to the metadata inside the script allows for some extra validations to be done in particular when moving NFTs around.
```
Strangely "put a metadata in the data room" actually makes sense here 🤣